### PR TITLE
RPM Linux Distros

### PR DESCRIPTION
**Pull request by kaeptnkrunch**
added the commands for RPM based Linux distributions like Fedora/Red Hat and the File SDL_mixer.ymp for openSUSE.

### CODE REVIEW

**Review by ihhub — COMMENTED** [review]
Hi @kaeptnkrunch could you please check given comments?

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @kaeptnkrunch could you please take a look into given comments?

File: script/linux/SDL_mixer.ymp [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      <repository recommended="true">
+        <name>openSUSE:Leap:15.2:Update</name>
+        <summary>Online updates for openSUSE Leap:15.2</summary>
+        <description>This project is releasing the official updates for openSUSE Leap:15.2.</description>
+        <url>http://download.opensuse.org/update/leap/15.2/oss/</url>
+      </repository>
+      <repository recommended="true">
+        <name>openSUSE:Leap:15.2:NonFree</name>
+        <summary>Non Free parts of openSUSE Leap 15.2 distribution.</summary>
+        <description>Non free or non open source parts of openSUSE Leap 15.2.</description>
```
[the remark this answers is not in the document]
Comment (kaeptnkrunch) [reply script/linux/SDL_mixer.ymp:19]: Hey, yes its a File for One-Click-Install of the SDL. Its common in openSUSE and beginner Friendly

**Comment by ihhub** [comment]
@kaeptnkrunch thank you very much for you contribution to the project!

**Comment by kaeptnkrunch** [comment]
I´m sry for my late reply

**Pull request merged**